tstest/natlab/vnet/conf.go

```diff
@@ -141,7 +142,11 @@ func (c *Config) AddNetwork(opts ...any) *Network {
 			if ip, err := netip.ParseAddr(o); err == nil {
 				n.wanIP = ip
 			} else if ip, err := netip.ParsePrefix(o); err == nil {
-				n.lanIP = ip
+				if ip.Addr().Is4() {
+					n.lanIP = ip
+				} else if ip.Addr().Is6() {
+					n.wanIP6 = ip
```

Comment:
Hmm, leave a comment here that the wantIP6 is deliberate and not a typo. The if block has lanIP

Reply:
done, and renamed some fields